odoo/tools/translate.py

```diff
@@ -1146,17 +1146,17 @@ def _export_translatable_resources(self):
                 for fname in fnmatch.filter(files, '*.py'):
                     self._babel_extract_terms(fname, path, root,
                                               extract_keywords={'_': None, '_lt': None})
-                # Javascript source files in the static/src/js directory, rest is ignored (libs)
-                if fnmatch.fnmatch(root, '*/static/src/js*'):
+                if fnmatch.fnmatch(root, '*/static/src*'):
+                    # Javascript source files in the static/src/js directory
```

Comment:
... and all over the place :partying_face: 

```suggestion
                    # Javascript source files
```